client/public/index.html

```diff
@@ -24,6 +24,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
+    rel="stylesheet"
+    href="https://unpkg.com/flickity@2/dist/flickity.min.css"
```

Comment:
The team agreed upon using bootstrap as the CSS framework.

Reply:
I cannot use Flickity?

Reply:
We're focusing on speed and every new framework we'll be adding will result in decreasing the speed.
Moreover, it can be easily implemented in react thanks to the components feature.

Reply:
Bootstrap is already installed. There are some libraries that attempt to develop an implementation of Bootstrap that is specific to React that will allow us to use JSX components as well as work with Bootstrap styles. 
So, I have to install like reactstrap or there is an alternative that I use the CDN directly in index.html.
Which is more suitable?

Reply:
Yes you can install reactstrap using NPM! I think we agreed on this, @EmperorYP7 ?

Reply:
react-bootstrap is preferred over reactstrap.